src/test/java/org/phoebus/channelfinder/docker/ChannelFinderTagsIT.java

```diff
@@ -129,6 +133,28 @@ public static void tearDownObjects() {
         tag_t1_owner_o2 = null;
     }
 
+    @AfterAll
+    public static void extractJacocoReport() {
+        // extract jacoco report from container file system
+        //     stop jvm to make data available
+
+        if (!Boolean.FALSE.toString().equals(System.getProperty(ITUtil.JACOCO_SKIPITCOVERAGE))) {
+            return;
+        }
+
```

Comment:
This method is repeated in all the docker IT Tests. It could be written once in ITUtil and called with the arguement of the class simple name.

Reply:
Done. Also to be handled in similar way is ENVIRONMENT, i.e. compose container.